docs/source/config/users.md

```diff
@@ -14,3 +14,18 @@ const options = {
   }
 };
 ```
+### Storing additional values
+
+The standard user data type stores most of the information a user could have. But if you want to store additional information in user, you can do so by extending `UserProfileInput` that is passed to the `Mutation.updateUserProfile` to store the additional fields you want :
+
+```
+extend input UserProfileInput {
```

Comment:
I'm not sure if this is the right approach, if you look at how meta is changed in other mutations, it looks like this:

```
input UpdateProductInput {
      tags: [String!]
      sequence: Int
      meta: JSON
    }
```

So the input is not extended but there is a meta field in the input, only the type to read from is extended.